transport_test.go

```diff
@@ -130,23 +129,21 @@ func TestGetRequestBodyFromEventCompletelyInvalid(t *testing.T) {
 	}
 }
 
-func TestGetEnvelopeFromBody(t *testing.T) {
-	body := getRequestBodyFromEvent(&Event{
-		Type:           transactionType,
-		Spans:          []*Span{},
-		StartTimestamp: time.Unix(3, 0).UTC(),
-		Timestamp:      time.Unix(5, 0).UTC(),
-	})
-	env := getEnvelopeFromBody(body, time.Unix(6, 0))
-	got := env.String()
-	//nolint: lll
-	want := strings.Join([]string{
-		`{"sent_at":"1970-01-01T00:00:06Z"}`,
-		`{"type":"transaction"}`,
-		`{"sdk":{},"timestamp":"1970-01-01T00:00:05Z","user":{},"type":"transaction","start_timestamp":"1970-01-01T00:00:03Z"}`,
-	}, "\n")
+func TestTransactionEnvelopeFromBody(t *testing.T) {
+	const eventID = "b81c5be4d31e48959103a1f878a1efcb"
+	sentAt := time.Unix(0, 0).UTC()
```

Comment:
Do you use `UTC` on purpose here? Because a default for `transactionEnvelopeFromBody` is not using it.

Reply:
We need either UTC or any other fixed location, otherwise the test output will depend on the time zone of the computer running the test.